netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -170,6 +170,9 @@ public final NettyChannelBuilder eventLoopGroup(@Nullable EventLoopGroup eventLo
    * GrpcSslContexts}, but options could have been overridden.
    */
   public final NettyChannelBuilder sslContext(SslContext sslContext) {
+    if (sslContext != null && sslContext.isServer()) {
```

Comment:
You can use the checkArgument() function here which can be statically imported from Preconditions.